src/reduxStore/missions.js

```diff
@@ -18,11 +18,33 @@ export const getMissions = createAsyncThunk(GET_DATA, async () => {
 
 const missionsSlice = createSlice({
   name: 'missions',
-  initialState: [{ mission_name: 'Rocket1', mission_id: 1, description: 'Lorem' }],
+  initialState: [{
```

Comment:
Here I can see you have initialized the state twice, you can remove one so that your code can conform to the dry principle